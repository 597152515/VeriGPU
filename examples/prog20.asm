li x1, -2.5
outr.s x1
li x1, 1.23
outr.s x1
li x1 123.456
outr.s x1
li x1 0.000123456
outr x1
outr.s x1
halt
