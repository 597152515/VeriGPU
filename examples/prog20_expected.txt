out.s 0 11000000001000000000000000000000 -2.500000
out.s 1 00111111100111010111000010100011 1.230000
out.s 2 01000010111101101101100101111000 123.424744
out 3 390173f7 956396535
out.s 4 00111001000000010111001111110111 0.000123